### PR TITLE
Manually updating the build version to `1.3.3.0`.

### PR DESCRIPTION
**Pull request by tannergooding**
FYI. @Pilchie, @jasonmalinowski, @MattGertz 

@jasonmalinowski has a proper fix against master here: https://github.com/dotnet/roslyn/pull/12151

This is required to unblock internal PR 1134

### CODE REVIEW

File: build/Targets/VSL.Versions.targets [hunk 1]
```diff
@@ -58,7 +58,7 @@
   </Choose>
 
   <!-- Returns the current build version. Used in .vsixmanifests to substitute our build version into them -->
-  <Target Name="GetBuildVersion" Outputs="$(BuildVersion)" />
+  <Target Name="GetBuildVersion" Outputs="1.3.2.0" />
```
Comment (jasonmalinowski) [inline build/Targets/VSL.Versions.targets]: This should be 1.3.3 since MicroUpdate 1 was 1.3.2.

Comment (tannergooding) [inline build/Targets/VSL.Versions.targets]: Fixed.

**Comment by Pilchie** [comment]
:+1:

**Comment by jasonmalinowski** [comment]
:+1:

**Pull request merged**